lib/options.cpp

```diff
@@ -15,7 +15,8 @@ limitations under the License.
 */
 
 #include "options.h"
-#include <lib/null.h>
+
+#include "null.h"
```

Comment:
I was mainly surprised this failed in `bazel`, but not everywhere else. This seems like some sort of inconsistency in the build system, but since I don't know bazel I didn't want to investigate it.

Reply:
Ah, the quotes in my comment got swallowed.

The lack of support for angled brackets is by design: https://bazel.build/docs/bazel-and-cpp#include-paths